src/pfe/file-watcher/server/test/scripts/setup.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+# Colors for success and error messages
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+BLUE='\033[0;36m'
+RESET='\033[0m'
+
+CW_DIR=~/codewind
+CODEWIND_REPO=git@github.com:eclipse/codewind.git
+
+function usage {
+    me=$(basename $0)
+    cat <<EOF
+Usage: [-<option letter> <options value> | -h]
+Options:
+    -t # Test type, currently support 'local' and 'kube' - Mandatory
+    -b # Test branch - Optional
+    -f # Setup function, currently support 'install' and 'uninstall' - Mandatory
+    -h # Display the man page
+EOF
+}
+
+# Install function does the following things
+# 1 Download Codewind repository
+# 2 Build Codewind images
+# 3 Run Codewind containers
+function install {
```

Comment:
I'm not sure we need an install/uninstall based on the previous comment about reusing the existing repo instead of cloning a separate one.